src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 
 import style from './dialogs.module.css';
+import I18Y from '../../Core/constants';
 
 const Dialogs = () => (
-  <div className={style.profile}>
-    Dialogs ve been here
+  <div className={style.dialogs}>
+    <div className={style.dialogsItems}>
+      <div className={style.dialog}>{I18Y.name[0]}</div>
```

Comment:
Consider re-writing this JSX to be a function getNames that map through names array and applies a getName function.
getName function should receive a text name and return an appropriate div element.

Reply:
I added new jsx file to fix this problem